README.md

```diff
@@ -14,30 +14,69 @@ an exact solution if possible or use an appropriate numeric solver otherwise.
 
 `src` - The source code of NESTML
 
-## Docker
-The docker files can be find in the `docker` folder.
+## Installing and running NESTML
 
-For the usage:
-Docker must be installed on the machine. See the follwing resource for the installation instructions https://docs.docker.com/engine/installation
-Clone the github repository locally. Switch to the docker-folder in the nestml repository.
+To install NESTML, the following requirements need to be met:
+* [Apache Maven](https://maven.apache.org/)
```

Comment:
@Silmathoron: good suggestion. However, I'd like to refrain from putting too much detail information into the README.

We're currently in the last phase of writing the first version of our reference manual (see https://github.com/jougs/nestml/tree/reference_documentation and https://github.com/jougs/nestml/pull/3) I'm happy to include more detailed instructions there, also pointing out other means of installation for the dependencies.

Feel free to create a corresponding issue with text suggestions or wait until https://github.com/jougs/nestml/pull/3 is merged and suggest them by a PR.